pyuvdata/parameter.py

```diff
@@ -711,6 +713,8 @@ class LocationParameter(UVParameter):
        Always set to 3.
     description : str
         Description of the data or metadata in the object.
+    frame : str, optional
+        Coordinate frame. Valid options are "itrs" (default) or "mcmf".
```

Comment:
Is this restriction (to "itrs" or "mcmf") enforced anywhere?

Reply:
It looks like it's implicitly true because of the way you're setting the `acceptable_range`, but I don't see it being tested explicitly. Another approach would be to just not set the acceptable range if it's not "itrs" or "mcmf".